lms/static/js/jquery.gradebook.js

```diff
@@ -27,7 +27,8 @@ var Gradebook = function($element) {
 
 	var moveDrag = function(e) {
 		var offset = e.pageX - mouseOrigin;
-		var targetLeft = clamp(tableOrigin + offset, -maxScroll, 0);
+		// Here updating direction of scroll.
```

Comment:
This comment isn't clear. Are you commenting the change you made, or the code as it exists after the change? Comments in the code should not address changes, like why you changed a negative value to a positive value. Comments should explain what the code is *currently* doing. The idea is that, if I understand what the code is supposed to be doing, then I can understand why the negative value was wrong, and it should be a positive value.